app/scripts/api.js

```diff
@@ -682,6 +683,11 @@ const createApi = function api(context, pubSub) {
             apiPubSub.unsubscribe('mouseMoveZoom', callback);
             break;
 
+          case 'wheel':
+            self.numWheelCallbacks -= 1;
```

Comment:
I wonder if there's a way to just count the number of listeners in apiPubSub instead of maintaining another variable to keep track of that.

Reply:
Right now only the `publish` method is actually available in HiGlassComponent https://github.com/higlass/higlass/blob/develop/app/scripts/api.js#L32, so I will work on trying to make the listeners or number of listeners available as well

Reply:
OK, it's not essential but I think it would reduce a little bit of complexity.